runtime/verbose/errormessagehelper.c

```diff
@@ -889,8 +889,8 @@ mapDataTypeToUTF8String(J9UTF8Ref* dataType, StackMapFrame* stackMapFrame, Metho
 
 			dataType->bytes = methodInfo->signature.bytes + typeValue;
 			dataType->length = typeLength;
-			/* Ignore 'L' and ';' to get the full string of argument in signature */
-			if ('L' == *dataType->bytes) {
+			/* Ignore 'L', 'Q', and ';' to get the full string of argument in signature */
+			if (IS_REF_OR_VAL_SIGNATURE(*dataType->bytes)) {
```

Comment:
While you're here, please fix the preprocessor lines 654 & 658: `#` should be at the beginning of the line and `#if` should be omitted from the `#endif` comment.

Reply:
I've noticed there are a lot more places in the codebase that have these inconsistencies in both C and Java preprocessor comments (most of which are my fault). I could fix all of them in a separate PR if you think it's worthwhile? Alternatively I could fix them in this PR (would only be about 40 more lines changed but it would touch files that haven't been changed yet in this PR).

Reply:
I think it's reasonable to fix those other files in this pull request.

Reply:
I fixed all the existing inconsistent C preprocessor comments in [6d7788e](https://github.com/eclipse-openj9/openj9/commit/6d7788e4b8c90054c809fd5ae595d6528140f68d), but not all the Java ones because there turned out to be way more than I thought (around 150 instances). Is that still reasonable for this PR do you think?

Reply:
Thoughts on this @keithc-ca?

Reply:
I'm happy to defer a cleanup of those Java source files.